sql/core/src/test/resources/sql-tests/results/cte.sql.out

```diff
@@ -89,16 +89,25 @@ struct<c1:int,c2:int>
 
 
 -- !query 7
-DROP VIEW IF EXISTS t
+WITH t(x) AS (SELECT 1)
+SELECT * FROM t WHERE x = 1
```

Comment:
Almost all the test cases are using one column in the CTE definition. Can you try your best to improve the current test coverage in this new syntax?